docs/sources/awssns.md

```diff
@@ -1,126 +1,105 @@
 
 ```console
-$ kubectl -n <my_namespace> create -f my-awssns-containersource.yaml
+$ aws --region us-west-2 sns list-topics
+{
+    "Topics": [
+        ...
+        {
+            "TopicArn": "arn:aws:sns:us-west-2:123456789012:MyTopic"
+        },
+        ...
+    ]
+}
 ```
 
-### As a Deployment object bound by a SinkBinding
+### API Credentials
+
+The TriggerMesh AWS SNS event source authenticates calls to the AWS SNS API using an [Access Key][accesskey]. The page at this link contains instructions to create an access key when signed either as the root user or as an IAM user. Take note of the **Access Key ID** and **Secret Access Key**, they will be used to create an instance of the event source.
+
+It is considered a [good practice][iam-bestpractices] to create dedicated users with restricted privileges in order to programmatically access AWS services. Permissions can be added or revoked granularly for a given IAM user by attaching [IAM Policies][iam-policies] to it.
+
+As an example, the following policy contains the permissions required by the TriggerMesh AWS SNS event source to read and delete messages from any topic linked to the AWS account:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "sns:Subscribe",
+                "sns:ConfirmSubscription"
+            ],
+            "Resource": "arn:aws:sns:*:123456789012:e2e-*"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": "sns:Unsubscribe",
+            "Resource": "*"
+        }
+    ]
+}
+```
 
-Copy the sample manifest from `config/samples/awssns-sinkbinding.yaml` and replace the pre-filled environment variables
-under `env` with the values corresponding to your _AWS SNS_ topic. Then, create the `Deployment` and `SinkBinding`
-objects in your Kubernetes cluster:
+![Creating an IAM user](../images/awssns-source/sns-user-policy.png)
 
-```console
-$ kubectl -n <my_namespace> create -f my-awssns-sinkbinding.yaml
-```
+## Deploying an Instance of the Source
 
-## Running locally
+Open the Bridge creation screen and add a source of type `AWS SNS`.
 
-Running the event source on your local machine can be convenient for development purposes.
+![Adding an AWS SNS source](../images/awssns-source/bridge-form-sns-source.png)
 
-### In the shell
+In the Source creation form, give a name to the event source and add the following information:
 
-Ensure the following environment variables are exported to your current shell's environment:
+* [**Secret**][accesskey]: Reference to a [TriggerMesh secret][tm-secret] containing an Access Key ID and a Secret Access Key to communicate with the AWS SNS API, as described in the previous sections.
+* [**AWS ARN**][arn]: ARN of the SNS topic, as described in the previous sections.
```

Comment:
the screenshot also has an entry for `DeliveryPolicy`, should that be documented ?

Reply:
it comes from the bridge template

Reply:
done!